src/preprocessor_output/factor_solve_utilities.F90

```diff
@@ -94,38 +94,40 @@ subroutine factor(LU)
 
   real(r8), intent(inout) :: LU(:,:)
 
-  integer :: i
-
-  do i = 1, ncell
-      LU(i,1) = 1./LU(i,1)
-      LU(i,2) = LU(i,2) * LU(i,1)
-      LU(i,3) = LU(i,3) * LU(i,1)
-      LU(i,4) = LU(i,4) * LU(i,1)
-      LU(i,5) = 1./LU(i,5)
-      LU(i,6) = 1./LU(i,6)
-      LU(i,7) = 1./LU(i,7)
-      LU(i,8) = 1./LU(i,8)
-      LU(i,9) = LU(i,9) * LU(i,8)
-      LU(i,10) = LU(i,10) * LU(i,8)
-      LU(i,11) = 1./LU(i,11)
-      LU(i,12) = LU(i,12) * LU(i,11)
-      LU(i,17) = LU(i,17) - LU(i,12)*LU(i,16)
-      LU(i,21) = LU(i,21) - LU(i,12)*LU(i,20)
-      LU(i,13) = 1./LU(i,13)
-      LU(i,14) = LU(i,14) * LU(i,13)
-      LU(i,15) = LU(i,15) * LU(i,13)
-      LU(i,18) = LU(i,18) - LU(i,14)*LU(i,17)
-      LU(i,19) = LU(i,19) - LU(i,15)*LU(i,17)
-      LU(i,22) = LU(i,22) - LU(i,14)*LU(i,21)
-      LU(i,23) = LU(i,23) - LU(i,15)*LU(i,21)
-      LU(i,18) = 1./LU(i,18)
-      LU(i,19) = LU(i,19) * LU(i,18)
-      LU(i,23) = LU(i,23) - LU(i,19)*LU(i,22)
-      LU(i,23) = 1./LU(i,23)
+integer :: i
```

Comment:
It's up to you - this is all just for a test, so I'm ok with however you'd like to structure things

Reply:
I see. Could you please help revert the files under the **src** folder back to what they are under the `main` branch? I feel confused about the `src` and `test` folders until I look at the dockerfile.

Reply:
sure thing! this is because of yet another problem with preprocessors. In MusicBox, we don't always use MICM, so we have "empty" functions in the `src/preprocessor_output/` folder so it compiles even when there is no mechanism chosen. But for testing (which doesn't really exist yet), we included the Chapman versions of the preprocessor output in the `test/preprocessor_output/` folder. I'll revert the `src/preprocessor_output/` back to `main`

Reply:
Thanks for the detailed explanation. That is very helpful. I will remember to only change the `test/preprocessor_output` in the future work.